health/health.d/dns_query.conf

```diff
@@ -11,5 +11,6 @@ component: DNS
     every: 10s
      warn: $this == nan
     delay: up 20s down 5m multiplier 1.5 max 1h
-     info: average DNS query round trip time over the last 10 seconds
+     info: Average DNS query round trip time over the last 10 seconds. \
```

Comment:
This line is actually not technically correct. It should properly indicate that there was a failure querying the server.

Reply:
This line describes the value, which is RTT. We are using the lack of the value (`nan`) as an indication of the problem (_Failed to query the DNS server_).